server/fishtest/templates/actions.mak

```diff
@@ -60,11 +60,11 @@
             <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
           </div>
           <div class="modal-body text-break">
-            This will perform a case insensitive free text search on the fields event name, user name, worker name,
+            This will perform a case insensitive text search on the fields event name, user name, worker name,
             target user, network name, branch name and comment.
-            In most cases one can only search for a list of full words, i.e. strings bounded by delimiters.
+            One can only search for a list of full words, i.e. strings bounded by delimiters.
```

Comment:
> For example the worker name chessuser-128cores-abcdefgh-uvwx matches **abcdefh** but not abcde.

If I'm not wrong should be "abcdefgh", reported by salvo_sf-fan on discord

Reply:
Fixed!